sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -490,6 +516,17 @@ export interface FileItem {
     properties: FileProperty;
 }
 
+// @public
+export class FileLeaseClient {
```

Comment:
Better to name it `ShareFileLeaseClient`

Reply:
Or `ShareLeaseClient`, depending on what is agree in the discussion email thread.

Reply:
Will do.